gocourse04/zoo/sector.go

```diff
@@ -0,0 +1,38 @@
+package zoo
+
+import "fmt"
+
+type Sector struct {
+	Subtype string
+	Animals []Animal
+}
+
+func NewSector(subtype string) *Sector {
+	return &Sector{
+		Subtype: subtype,
+		Animals: make([]Animal, 0),
+	}
+}
+
+func (s Sector) AddAnimal(animal *Animal) Sector {
```

Comment:
Чи можемо ми не повертати Sector, змінивши на:

```suggestion
func (s *Sector) AddAnimal(animal *Animal) {
```

?